### PR TITLE
Add r-doubletfinder recipe

### PR DESCRIPTION
**Pull request by nictru**
This PR adds the [DoubletFiner](https://github.com/chris-mcginnis-ucsf/DoubletFinder) R package

### CODE REVIEW

**Comment by nictru** [comment]
@BiocondaBot please add label

File: recipes/r-doubletfinder/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+package:
+  name: r-doubletfinder
+  version: 1.0.0
+
+source:
+  url: https://github.com/chris-mcginnis-ucsf/DoubletFinder/archive/03e9f37f891ef76a23cc55ea69f940c536ae8f9f.zip
```
Comment (martin-g) [inline recipes/r-doubletfinder/meta.yaml:6]: Any chance you could ask the author to make a release/tag ?

File: recipes/r-doubletfinder/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  commands:
+    - $R -e "library(DoubletFinder)"
+
+about:
+  home: https://github.com/chris-mcginnis-ucsf/DoubletFinder
+  license: unknown
+  summary: R package for detecting doublets in single-cell RNA sequencing data
+extra:
+  additional-platforms:
+    - linux-aarch64
```
Comment (martin-g) [inline recipes/r-doubletfinder/meta.yaml:41]: `additional-platforms` is not needed for `noarch` recipes

**Review by martin-g — CHANGES_REQUESTED** [review]

File: recipes/r-doubletfinder/meta.yaml (hunk 1 above)
Comment (nictru) [reply recipes/r-doubletfinder/meta.yaml:6]: I could, but would probably take a while. 
Would it be an acceptable intermediary  solution to have it as a 0.0.1 version until we have a proper tag?

File: recipes/r-doubletfinder/meta.yaml (hunk 2 above)
Comment (nictru) [reply recipes/r-doubletfinder/meta.yaml:41]: Ah, good to know, thanks

**Pull request closed without merging**